### PR TITLE
Add WH portion in Aboutus md

### PR DESCRIPTION
**Pull request by WeeeHung**
added my portion of about us md

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[homepage](https://github.com/WeeeHung)]
+[[github](https://github.com/WeeeHung)]
+[[portfolio](team/weeehung.md)]
 
-* Role: Project Advisor
+* Role: Peasant
+* Responsibilities: Anything the team wants me to do
```
Comment (WZWren) [inline docs/AboutUs.md:23]: Are you sure you want to add this to the main branch?

Comment (WeeeHung) [reply docs/AboutUs.md]: I've updated my new roles and responsibilities!

**Review by WZWren — CHANGES_REQUESTED** [review]
I think something went wrong with your file structure - you've accidentally pushed your the binaries into the branch.

**Review by WZWren — APPROVED** [review]
This looks fine now, merging!

**Pull request merged**